js/app.js

```diff
@@ -1,3 +1,107 @@
-'use strict'
+'use strict';
 
-console.log('On the other side of the screen, it all looks so easy.');
+//console.log('On the other side of the screen, it all looks so easy.');
+
+// Global variables
+let voteCount = 0; // user's product vote counter
+let maxVotes = 5; // maximum number of product voting rounds
+let displayQty = 3; // number of products to be displayed at one time for voting
+let prodObjs = []; // arrary of product objects
+let allProducts = [ // eslint-disable-line
```

Comment:
This array is never used, when you could be using it instead of prodObjs. I would recommend removing one.